client/src/components/Users/index.jsx

```diff
@@ -305,6 +306,7 @@ class Users extends Component {
                 rowId={record.id}
                 users={this.state.users}
                 handleDeleteUser={this.handleDeleteUser}
+                changePassword = {() => this.handleChangePassword('Change')}
```

Comment:
@AymanAlqoqa This isn't super important, but this component will be more efficient (use less memory) if you can remove the inline function definitions (the `() => {}`) from render.  Every time render() is called (which can happen frequently), the `() => {}` is re-run, so another copy of that function is created in memory.
Instead, you could add a small function to this class:
```
handleChangePasswordChangeArg = () => { /* I think I need to come up with a better name */
  this.handleChangePassword('change')
}
```
And then use it here:
`changePassword = {this.handleChangePasswordChangeArg}`